tst/construct.tst

```diff
@@ -17,8 +17,13 @@ gap> ConstructAllCFGroups(0);
 Error, Argument has to be a positive cube-free integer.
 gap> ConstructAllCFGroups(1);
 [ <pc group of size 1 with 0 generators> ]
-gap> ConstructAllCFGroups(2);
-[ <pc group of size 2 with 1 generators> ]
+gap> x := ConstructAllCFGroups(2);;
+gap> Length(x) = 1 and IsPcGroup(x[1]) and Size(x[1]) = 2;
+true
+gap> CodePcGroup(x[1]);
+0
+gap> IdGroup(x[1]);
+[ 2, 1 ]
```

Comment:
I think just this would be sufficient:
```suggestion
gap> List(ConstructAllCFGroups(2), IdGroup);
[ [ 2, 1 ] ]
```
and likewise below.

This checks all the important bits: that the right number of groups with the right isomorphism type is returned. It doesn't check the number of generators or that those are pc groups, but that's (a) already checked by the other tests, (b) not really a concern we need to worry about here.